arbitrum-docs/launch-orbit-chain/how-tos/arbos-upgrade.md

```diff
@@ -16,17 +16,23 @@ import PublicPreviewBannerPartial from '../../node-running/partials/_upgrade-cad
 
 #### Step 1: Update Nitro on nodes and validators
 
-Refer to the release notes supporting the ArbOS version that you're upgrading to. You'll find a specific version of the Nitro software stack specified. For example, if your upgrade targets ArbOS 11, you'd use Nitro `v2.2.2` (Docker image: `offchainlabs/nitro-node:v2.2.2-8f33fea`) or higher. This is the version of the Nitro stack that needs to be running on each of your Orbit chain's nodes.
+Refer to the release notes supporting the ArbOS version that you're upgrading to. You'll find a specific version of the Nitro software stack specified. For example, if your upgrade targets ArbOS 11, you'd use Nitro `v2.3.0` (Docker image: `offchainlabs/nitro-node:v2.3.0-3e14543`) or higher. This is the version of the Nitro stack that needs to be running on each of your Orbit chain's nodes.
 
 Begin by upgrading your validator node(s) to the specified Nitro version, then update each remaining Orbit node to match this version.
 
 Note that this update _must occur_ before the deadline established for the target ArbOS upgrade. Refer to the timestamp in the ArbOS upgrade schedule for a precise deadline.
 
-#### Step 2: Upgrade the Wasm module root
+#### Step 2: Upgrade the Wasm module root & your chain's Nitro contracts
 
-The `WASM module root` is a 32-byte hash, created from the Merkelized Go replay binary and its dependencies. When ArbOS is upgraded, a new Wasm module root is generated due to modifications in the State Transition Function. This new Wasm module root must be set in the rollup contract on the parent chain. To set the new Wasm module root, use the `Rollup proxy` contract's [setWasmModuleRoot](https://github.com/OffchainLabs/nitro-contracts/blob/38a70a5e14f8b52478eb5db08e7551a82ced14fe/src/rollup/RollupAdminLogic.sol#L321) method.
+To update the Wasm module root and deploy your chain's Nitro contracts to the parent chain for the most recent ArbOS release, please [follow the instructions in this guide](https://github.com/OffchainLabs/orbit-actions/tree/main/scripts/foundry/contract-upgrades/1.2.1). Note that each ArbOS release will require performing this step with a different Wasm module root and a different version of `nitro-contracts`. The guide linked above will be kept updated with the instructions for each specific ArbOS release.
```

Comment:
There may or may not be a contract upgrade associated with a nitro upgrade I'd say (unless you count just changing the wasm module root as a contract upgrade). Also the structure of the orbit actions repo currently has that page linked to being instructions for just this specific upgrade with the intent being to have a different page with instructions on the next upgrade. That's certainly something that could be reconciled with changes on either side